docs/source/api/python/nn-pytorch.rst

```diff
@@ -43,6 +41,11 @@ Conv Layers
     ~dgl.nn.pytorch.conv.PNAConv
     ~dgl.nn.pytorch.conv.DGNConv
 
+CuGraph Conv Layers
+----------------------------------------
```

Comment:
Add an empty line below and add

```
.. autosummary::
    :toctree: ../../generated/
    :nosignatures:
    :template: classtemplate.rst
```

Reply:
Ah, forgot to code over this snippet. Added now.